### DISCUSSION TITLE
Saving Data

### DISCUSSION THREAD

**Discussion by KamlaXPL** [OmerBenGera/WildStacker#543] [Implemented Feature Requests] [answered]
Saving things to the database, because a crash or whatever causes data loss, the most important thing is spawners and barrels to save, because undoing spawners and barrels pisses players off the most

**Comment by OmerBenGera** [comment, marked as answer]
Spawners and barrels are saved into db when changed whuile entity and item amounts are saved as an extra data attached to the entities.
